federation/cluster/federation-up.sh

```diff
@@ -114,10 +114,9 @@ function init() {
       --dns-zone-name="${DNS_ZONE_NAME}" \
       --dns-provider="${DNS_PROVIDER}" \
       --image="${kube_registry}/hyperkube-amd64:${kube_version}" \
-      --apiserver-arg-overrides="--storage-backend=etcd2" \
       --apiserver-enable-basic-auth=true \
       --apiserver-enable-token-auth=true \
-      --apiserver-arg-overrides="--v=4" \
+      --apiserver-arg-overrides="--runtime-config=api/all=true,--v=4" \
```

Comment:
Thanks @nikhiljindal for having a look. Actually this discussion already happened quite some time back as in [this comment on this pr](https://github.com/kubernetes/kubernetes/pull/46358#discussion_r118604152) and [this code comment](https://github.com/kubernetes/kubernetes/blob/master/federation/cluster/federation-up.sh#L21). Please let me know your comments.